app/res/layout/home_screen.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Not really a big deal and definitely not worth changing if it proves at all annoying, but is there a reason this has to be 4 levels of nested layouts, when only the RecyclerView actually has content?